src/RefreshAuthStateController.js

```diff
@@ -27,10 +27,12 @@ define(['q', 'okta', 'util/FormController'], function (Q, Okta, FormController)
       var token = this.options.token;
       this.model.startTransaction(function (authClient) {
         appState.trigger('loading', true);
-        if (token && (token === this.settings.get('stateToken'))) {
-          // widget bootstrapped with statetoken via settings
-          // check if stateToken in url is same as settings
-          //unset stateToken to prevent the baseloginrouter controller from calling it again on render
+        if (token && (token === this.settings.get('stateToken') ||
+          this.options.appState.get('introspectSuccess'))) {
```

Comment:
wondering if `this.options.appState.get('introspectSuccess')` alone is sufficient ?

Reply:
yeah, I was hesitant to remove the other check which was added to fix a mobile flows, that hits the widget with a stateToken in the url.

Reply:
We will have to check for `this.options.appState.get('introspectSuccess') || this.options.appState.get('introspectError')` otherwise in case of an error during introspect, it'll make another request to authn API.